integration-tests/ccip-tests/testreporters/ccip.go

```diff
@@ -77,19 +77,17 @@ func (stat *RequestStat) UpdateState(
 	step Phase,
 	duration time.Duration,
 	state Status,
-	sendTransactionStats ...TransactionStats,
+	sendTransactionStats *TransactionStats,
```

Comment:
Nit: I find naming the argument as optional can help clarify things in cases like this.
```suggestion
	optionalSendTransactionStats *TransactionStats,
```